packages/hoprd/src/main.ts

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+
+// Replace default process name (`node`) by `hoprd`
+process.title = 'hoprd'
+
+const majorNodeVersion = process.version.match(/(?<=^v)[0-9]{1,}/)
+
+if (!majorNodeVersion || majorNodeVersion.length == 0 || parseInt(majorNodeVersion[0]) < 16) {
```

Comment:
We could read out the minimal version from `package.json` at `.engine.node` to remove duplication. But not required.